lib/Runtime/Library/WebAssemblyInstance.cpp

```diff
@@ -257,9 +257,21 @@ Var WebAssemblyInstance::CreateExportObject(WebAssemblyModule * wasmModule, Scri
             case Wasm::ExternalKinds::Memory:
                 obj = env->GetMemory(wasmExport->index);
                 break;
-            case Wasm::ExternalKinds::Function:
+            case Wasm::ExternalKinds::Function: {
                 obj = env->GetWasmFunction(wasmExport->index);
+                // Check whether there's any SIMD value in the function's signature.
+                Wasm::WasmSignature* sig = ((WasmScriptFunction*)(obj))->GetSignature();
```

Comment:
It is valid to export a function with simd arguments/return.
If that function is called from javascript then we should throw.
However, if that function is then imported in another wasm module it is fine to use.
In general, the handling of simd types in functions is fine as it is today. (unless it was spec'ed that way, but I don't see why)

Reply:
@Cellule 
Thanks for your comment. As arun is still on vacation, Would you please pend this issue and discuss with him when he's back?

Thanks!